applications/cuda_quantum/metadata.json

```diff
@@ -32,8 +32,8 @@
             "cuda_quantum": "^0.4.0"
         },
         "run": {
-            "command": "python3 cuda_quantum.py",
-            "workdir": "holohub_bin"
+            "command": "pip install -r requirements.txt && python3 cuda_quantum.py",
```

Comment:
I don't think we want pip install as part of the run. We should create a separate Dockerfile.

Reply:
See the PR description -- I also would prefer a Dockerfile, but `cuda_quantum` appears to have a transitive dependency on the [wheel-axle-runtime](https://pypi.org/project/wheel-axle-runtime/) package, which creates a conflict when running the app with different permissions than the installation environment. When I tried using a Dockerfile I saw permissions errors when running as a non-root user inside the container.

We could investigate further, but IMO it's worth making this exception to the rule for now to get the app working with `build_and_run`.